obspy/core/inventory/response.py

```diff
@@ -1551,7 +1551,7 @@ def approximation_type(self, value):
         self._approximation_type = value
 
 
-class FilterCoefficient(CustomFloat):
+class FilterCoefficient(FloatWithUncertainties):
```

Comment:
Huuuh. That's pretty strange, that coefficients have different types throughout stationxml.. and we're consistent in the confusion by using the dedicated `FilterCoefficient` type in this one place and `FloatWithUncertaintiesAndUnit` in other places for numerators/denominators..